packages/appium/scripts/postinstall.js

```diff
@@ -10,23 +23,15 @@ async function main() {
     return;
   }
 
-  let extension;
-  try {
-    extension = require('../build/lib/cli/extension');
-  } catch (e) {
-    throw new Error(
-      `Could not load extension CLI file; has the project been transpiled? ` + `(${e.message})`
-    );
-  }
-
-  const {DEFAULT_APPIUM_HOME, DRIVER_TYPE, PLUGIN_TYPE} = require('./build/lib/extension-config');
-  const {runExtensionCommand} = extension;
-  const appiumHome = process.env.npm_config_appium_home || DEFAULT_APPIUM_HOME;
   const specs = [
     [DRIVER_TYPE, driverEnv],
     [PLUGIN_TYPE, pluginEnv],
   ];
 
+  const appiumHome =
+    process.env.npm_config_appium_home || (await env.resolveAppiumHome());
```

Comment:
i *think* we can do away with this appium home npm install flag and just rely on APPIUM_HOME or the standard env discovery.

Reply:
OK.  Though ... I see a problem I hadn't thought of before.

This script is only applicable if Appium _isn't_ a direct dependency within the closest `package.json`. If that's the case, then it's expected these drivers/plugins will _also_ be direct dependencies.

I'll address this